macOS/DuckDuckGo-macOS.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 54;
+	objectVersion = 70;
```

Comment:
I think this will cause issues for older macOS CI runs as they rely on an older version of Xcode. That said, those runners were recently deprecated on the GitHub side, but we should probably revert the version change for now just in case we restore those tests.